### PR TITLE
Embellishes the vision exercise with more prompts

### PR DESCRIPTION
**Pull request by nickyspitz**
(inspired by the DE version)

### CODE REVIEW

File: VISION.md [hunk 1]
```diff
@@ -3,38 +3,41 @@
...
 
 * Lows:
 
 * Lessons Learned:
 
 
-### Conclude Step 1
+### Conclude the previous year
 
-* Looking at this list, what I intend to *take with me into next year* is:
```
Comment (ikiochoa) [inline VISION.md:16]: I don't find this so bad.

File: VISION.md [hunk 2]
```diff
@@ -3,38 +3,41 @@
...
 _Take no more than ten minutes to expound on each aspect of your life and give it a rating on a scale from one to ten, where ten is the best_
 
 _Pick your own topics to talk about, the following list (in alphabetical order) contains suggestions:_
 
+* Body Health
 * Career
 * Family
 * Friendships
+* Hobbies and Leisure
 * Health
```
Comment (ikiochoa) [inline VISION.md]: Perhaps you forgot to delete this

Comment (ikiochoa) [reply VISION.md]: And you might want to add finances to it

File: VISION.md [hunk 3]
```diff
@@ -3,38 +3,41 @@
...
 * Love
+* Mental Health
 * Personal Development
 * Philanthropy
 * Quality of Life
 * Sexuality
 * Spirituality
```
Comment (ikiochoa) [inline VISION.md:39]: In spanish I also added Emotions

File: VISION.md [hunk 4]
```diff
@@ -3,38 +3,41 @@
...
 
 ## Step 3: Create your vision for the future
-_This step is about working backwards from the "far-off" future to align with yourself as you continue to unfold. Take as much time as you need_
+_This step is about working backwards from the "far-off you" wish for yourself to align yourself for the coming year. For the prompts, take time to let your thoughts flow. Limit your time to 30 minutes if you think you'll go nuts. Then take a step back, perhaps drink a coffee or a glass of water. Come back to your vision and distill all of these thoughts into a motto for your upcoming year!_
```
Comment (ikiochoa) [inline VISION.md]: "far-off" you

File: VISION.md (hunk 3 above)
Comment (nickyspitz) [reply VISION.md:39]: I added 'emotional well-being'

**Comment by nickyspitz** [comment]
Thanks for the review, @ikiochoa 👍

**Pull request merged**